outputs.tf

```diff
@@ -2,3 +2,18 @@ output "static_ip" {
   description = "The static IP address attached to the VM instance."
   value       = local.static_ip
 }
+
+output "self_link" {
+  description = "The URI of the created resource."
+  value       = google_compute_instance.vm_instance.self_link
+}
+
+output "id" {
+  description = "an identifier for the resource with format projects/{{project}}/zones/{{zone}}/instances/{{name}}"
+  value       = google_compute_instance.vm_instance.id
+}
+
+output "instance_id" {
+  description = "The server-assigned unique identifier of this instance."
+  value       = google_compute_instance.vm_instance.instance_id
+}
```

Comment:
I like the example format given in the "id" description. Can we give similar example format for the others? Helps understand what am i to get out of this - when to use `self_link` vs when to use `id` vs when to use `instance_id` etc

Reply:
Sure, I'll update the descriptions of other TF modules.

Reply:
Not other TF modules. I meant the other new output attributes that you are introducing in this PR.